### ISSUE TITLE
Predict Tab Refresh

### ISSUE THREAD

**Issue by carldister** [Cschimpf/Complex-It#7]
Is there a way to put a refresh button in the PREDICT TAB?  i can run one file, but it will not upload a second file or allow me to run the predict function more than once.  so, i think it would be useful to have a REFRESH button, which would allow me to upload a new file without having to quit COMPLEX-IT, which is what I have to presently do. (from B)

**Comment by Cschimpf** [comment]
Yeah, it was originally written as a one-stream kind of thing, that people would be working on their one dataset throughout the complexit tabs because trying to control for trying to use all sorts of different data across different tabs becomes incredibly messy. I'm not sure what a safe assumption is if you go to a prediction tab and start using multiple data sets there, but I guess for now we can view predict tab as being a terminal tab for some?  I just want to be careful to not overwrite things they may have done in earlier tabs in case they want to go back there to review or save anything. 

Could you ask Brian to share its explicit behavior if gives when he uploads multiple files on the predict tab? I think what may be happening is the change isn't being pushed to through shiny's interactive engine and/or it may be some sort of error I need to catch/handle.

**Comment by carldister** [comment]
B agrees "So, let's not put in a REFRESH BUTTON." And, I agree that, as Corey says, I think what may be happening is the change isn't being pushed to through shiny's interactive engine and/or it may be some sort of error I need to catch/handle.  That seems to be more the issue that needs fixed.  It seems like something is running, but it is not refreshing on the screen.  So, if either of you do not mind, that would be something to check.  it is real easy to see, just run the predict tab and then try to upload a new database and it just freezes."
